podcastparser.py

```diff
@@ -674,8 +674,18 @@ def validate_episode(self):
         if len(entry['chapters']) == 0:
             del entry['chapters']
 
-        if 'description_html' in entry and not entry['description']:
-            entry['description'] = remove_html_tags(entry['description_html'])
+        # Ensures `description` and `description_html` are plain text and html if provided
+        if 'description' in entry and is_html(entry['description']):
+            if 'description_html' in entry:
+                if entry['description'] == entry['description_html']:
+                    entry['description'] = ''
+                else:
+                    logger.warning(
```

Comment:
If we are going to issue a warning (not sure if we should), it might be useful to have some context, like the URL or something, otherwise this warning will appear out of the blue with no info where this happens.